hdl/projects/sidecar/mainboard/test/PCIeEndpointControllerTests.bsv

```diff
@@ -50,7 +60,7 @@ module mkResetHostControlTest (Empty);
             "expected PCIe peripheral to not be reset");
     endseq);
 
-    mkTestWatchdog(20);
+    mkTestWatchdog(2500);
```

Comment:
Nit: I'd make this related to `pci_perst_tick_duration`, something like `mkTestWatchdog(pic_perst_tick_duration * 101 * 5)`, so things stay in sync. Even if the value of 101 were to change, a search for that value would show the relationship between those numbers.